Library/ViewModels/BackingViewModel.swift

```diff
@@ -124,66 +111,91 @@ public final class BackingViewModel: BackingViewModelType, BackingViewModelInput
-          timeStyle: .none
-      )
+    self.pledgeAmount = Signal.merge(
+      emptyStringOnLoad,
+      projectAndBackingAndBackerIsCurrentUser.map { project, backing, _ in
+        let basicPledge = backing.amount - (backing.shippingAmount ?? 0)
+        return Format.currency(basicPledge, country: project.country)
+      }
     )
-  }
-    self.backerPledgeAmountAndDateAccessibilityLabel = self.backerPledgeAmountAndDate.map { "Pledged " + $0 }
 
-    self.backerRewardDescription = Signal.combineLatest(project, reward)
-      .map { project, reward in
-        Strings.backer_modal_reward_amount_reward_description(
-          reward_amount: Format.currency(reward.minimum, country: project.country),
-          reward_description: reward.description
-        )
-    }
+    self.shippingAmount = Signal.merge(
+      emptyStringOnLoad,
+      projectAndBackingAndBackerIsCurrentUser
+        .map { project, backing, _ in
+          Format.currency(backing.shippingAmount ?? 0, country: project.country)
+      }
+    )
 
-    self.backerRewardDescriptionAccessibilityLabel = self.backerRewardDescription
+    self.totalPledgeAmount = Signal.merge(
+      emptyStringOnLoad,
+      projectAndBackingAndBackerIsCurrentUser.map { project, backing, _ in
+        Format.currency(backing.amount, country: project.country)
+      }
+    )
 
-    self.backerShippingDescription = reward.map { $0.shipping.summary  ?? "N/A" }
-    self.backerShippingDescriptionAccessibilityLabel = self.backerShippingDescription
+    self.statusDescription = Signal.merge(
+      emptyStringOnLoad.mapConst(NSAttributedString(string: "")),
+      projectAndBackingAndBackerIsCurrentUser
+      .map { project, backing, backerIsCurrentUser in
+        return statusDescString(for: backing, project: project, backerIsCurrentUser: backerIsCurrentUser)
+      }
+    )
 
-    self.backerShippingAmount = projectAndBackingAndBackerIsCurrentUser
-      .map { project, backing, _ in Format.currency(backing.shippingAmount ?? 0, country: project.country) }
-    self.backerShippingAmountAccessibilityLabel = self.backerShippingAmount
+    self.rewardSectionTitle = Signal.merge(
+      emptyStringOnLoad.map { NSAttributedString(string: $0) },
+      projectAndBackingAndBackerIsCurrentUser
+        .map { project, backing, backerIsCurrentUser in
+          rewardTitle(for: backing.reward, project: project, backerIsCurrentUser: backerIsCurrentUser)
+      }
+    )
 
-    self.estimatedDeliveryDateLabelText = reward.map {
-      $0.estimatedDeliveryOn.map {
-        Format.date(secondsInUTC: $0, dateFormat: "MMMM yyyy", timeZone: UTCTimeZone) }
+    self.rewardTitleWithAmount = Signal.merge(
+      emptyStringOnLoad,
+      projectAndBackingAndBackerIsCurrentUser
+        .map { project, backing, _ in
+          let currency = Format.currency(backing.reward?.minimum ?? 0, country: project.country)
+          if let rewardTitle = backing.reward?.title {
+            return currency + " - " + rewardTitle
```

Comment:
We get a lil more type-safety with `+` since it requires that the left- and righthand sides be strings. Interpolation can lead to some formatting gotchas.